examples/tensorflow/object_detection/main.py

```diff
@@ -123,6 +124,12 @@ def resume_from_checkpoint(checkpoint_manager, ckpt_path, steps_per_epoch):
     return initial_epoch, initial_step
 
 
+def extract_compression_state(ckpt_path: str):
+    checkpoint = tf.train.Checkpoint(compression_state=TFCompressionStateLoader())
+    load_checkpoint(checkpoint, ckpt_path)
+    return checkpoint.compression_state.state
```

Comment:
can I move to it a common part for TF samples?

Reply:
I think it's not necessary. For example, see `load_checkpoint()` method. All samples have their own copy.

Reply:
don't think it's a good practise. I had to modify all this copies and could make a mistake in some place. 
you have a common folder in tf samples. why not to move this helpers there?

Reply:
I discussed this moment with @alexsu52 and we came to the conclusion that all helpers for the sample should be inside one file. This is more understandable. But I agree, there is a problem with copy-paste and for this reason, it is difficult to support samples.